office365_admin/komand_office365_admin/actions/lookup_user_by_email/action.py

```diff
@@ -0,0 +1,42 @@
+import komand
+from .schema import LookupUserByEmailInput, LookupUserByEmailOutput, Input, Output, Component
+# Custom imports below
+import requests
+import json
+
+
+
+
+class LookupUserByEmail(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='lookup_user_by_email',
+                description=Component.DESCRIPTION,
+                input=LookupUserByEmailInput(),
+                output=LookupUserByEmailOutput())
+
+    def run(self, params={}):
+        email_address = params.get(Input.EMAIL_ADDRESS)
+        token = self.connection.access_token
+
+        headers = {"Authorization": "Bearer %s" % token, "Content-Type": "application/json"}
+        body_dict = {}
+
+        if not email_address:
```

Comment:
This should throw an exception (and I don't think it's possible, the tooling will throw a fit if there's no email address.) If it's not, make email_address a required parameter.

Reply:
done.  it is a required param